Client/mods/deathmatch/logic/luadefs/CLuaVehicleDefs.cpp

```diff
@@ -1291,6 +1293,31 @@ int CLuaVehicleDefs::GetHelicopterRotorSpeed(lua_State* luaVM)
     return 1;
 }
 
+std::variant<bool,float> CLuaVehicleDefs::GetVehicleRotorSpeed(CClientVehicle* pVehicle)
+{
+    float fSpeed = 0.0f;
+    // It's a plane or heli?
```

Comment:
I think it's pretty obvious what the `switch` statement does, so I would remove the comment explaining it.